tests/test_optional/test_px/test_px_input.py

```diff
@@ -14,8 +14,7 @@
 import warnings
 
 
-# FIXME: don't test with vaex if vaex isn't installed
-if (optional_imports.get_module("vaex") is None) and (sys.version_info >= (3, 12)):
+if optional_imports.get_module("vaex") is None:
```

Comment:
@marthacryan I would keep the upper bound here but just change it to complement what's in the `requirements.txt` so that if vaex randomly doesn't get installed when it should, the test will fail.

```suggestion
if optional_imports.get_module("vaex") is None and sys.version_info > (3, 9):
```